web/turkey/survey/static/survey/js/auditors/clicks_specific.js

```diff
@@ -0,0 +1,29 @@
+var AuditorClicksSpecifics = {
+    clicks_specifics: [],
+    log_click_content: function (e) {
+        e.stopPropagation();
```

Comment:
If DOM elements are layered, only the topmost element will be pushed to the array. For instance, if you click on a div, only that div will be pushed instead of both the div and the body.